src/web/components/elements/InteractiveBlockComponent.tsx

```diff
@@ -57,18 +63,171 @@ and is sent with all interactive elements in the scriptUrl from CAPI to do a two
 It has not been updated since 2016.
 
 MIGRATION FROM FRONTEND
-- TODO For the standard boot.js, we will re-write the behavior in modern JS to avoid the requirement of an AMD loader
+- For the standard boot.js, we have re-written the behavior in modern JS to avoid the requirement of an AMD loader
 and to avoid loading the boot.js file.
 - For all other files that do not load the standard boot.js, we'll add a AMD loader to the page
 
 For the remaining few we dynamically load and AMD loader and support the contract as defined with curl AMD loader
 
 */
+const supportingMinHeight = 200;
+const inlineMinHeight = 500;
+const getMinHeight = (role: string, loaded: boolean) => {
+	if (loaded) {
+		return `auto`;
+	}
+	return role === 'supporting'
+		? `${supportingMinHeight}px`
+		: `${inlineMinHeight}px`;
+};
+const wrapperStyle = (role: string, loaded: boolean) => css`
+	${body.medium()};
+	background-color: ${neutral[100]};
+	font-weight: 300;
+	min-height: ${getMinHeight(role, loaded)};
+	position: relative;
+`;
+
+const placeholderLinkStyle = css`
+	position: absolute;
+	bottom: 1rem;
+	left: 1rem;
```

Comment:
Are we doing rems now? I've been hearing talk about this but I'm still working to the assumption code should be written in pixels.

Reply:
I reach for rems by default, have updated!